solutions/bronze/usaco-1301.mdx

```diff
@@ -38,21 +38,19 @@ int main() {
 	vector<long long> days(n);
 	for (long long &d : days) { cin >> d; }
 
-	sort(days.begin(), days.end());
-
 	long long last_day = days[0];
 	long long cost = k + 1;  // Start the first subscription
-	for (long long d : days) {
+	for (int i = 0; i < n; i++) {
 		// Should Bessie extend the most recent subscription?
-		if (d - last_day < k + 1) {
-			cost += d - last_day;
+		if (days[i] - last_day < k + 1) {
+			cost += days[i] - last_day;
```

Comment:
not changed

Reply:
changed.
i thought that by just  changing the initial value to 0 would be enough.